test/unit/server-ha-active-service.bats

```diff
@@ -157,3 +157,53 @@ load _helpers
       yq -r '.spec.ports | map(select(.port==8200)) | .[] .name' | tee /dev/stderr)
   [ "${actual}" = "https" ]
 }
+
+@test "server/ha-active-Service: vault externalTrafficPolicy set to Local lowercase" {
```

Comment:
I think just testing that the value provided is honoured is good enough  here.  The two cases are:
- empty string `""`
- non empty value like `Cluster`

Reply:
@benashz So you suggest adding a default value of "Cluster" for instance ? 
I'm not handling the case where an empty string isn't provided.
This would probably fall back to K8S defaults (Which currently I'm not sure what is).

Reply:
@benashz Just checked - made a new svc with a value not in the correct case -  got the following error:
The Service "my-service" is invalid: spec.externalTrafficPolicy: Invalid value: "local": ExternalTrafficPolicy must be empty, Cluster or Local.

So - empty is just fine.
Non empty value would fail if it isn't Local or Cluster.